docs/src/snforge-advanced-features/storage-cheatcodes.md

```diff
@@ -160,4 +161,44 @@ fn store_in_complex_mapping() {
 > The `load` cheatcode will return zeros for memory you haven't written into yet (it is a default storage value for Starknet contracts' storage).
 
 
+## Example with `storage_address_from_base`
+
+This example uses `storage_address_from_base` with `address` function of the [storage variable](https://book.cairo-lang.org/ch14-01-contract-storage.html#addresses-of-storage-variables).
+
+To retrieve storage address of a given `field`, you need to import `{field_name}ContractMemberStateTrait`
+
+```rust
+#[starknet::contract]
+mod Contract {
+    #[storage]
+    struct Storage {
+        map: LegacyMap::<(u8, u32), u32>,
+    }
+}
+
+// ...
+use starknet::storage_access::storage_address_from_base;
+use snforge_std::{ store, load };
+use Contract::mapContractMemberStateTrait;
+
+#[test]
+fn update_mapping() {
+    let index = 1_u8;
+    let day = 10_u32;
+    let data = 42_u32;
```

Comment:
```suggestion
    let key = (1_u8, 10_u32);
    let data = 42_u32;
```
Imo it's more clear this way, `day` is ambiguous

Reply:
I agree, it's less ambiguous with key.